src/features/game/expansion/components/resources/tree/Tree.tsx

```diff
@@ -180,6 +180,7 @@ export const Tree: React.FC<Props> = ({ id }) => {
 
       {/* Chest reward */}
       <ChestReward
+        collectedItem={"Wood"}
```

Comment:
Just FYI, No need to wrap to wrap strings in `{}`.
```suggestion
        collectedItem="Wood"
```